### ISSUE TITLE
python-crypto obsoleted by python2-crypto

### ISSUE THREAD

**Issue by madAndroid** [echocat/puppet-graphite#243]
Hi,

We're experiencing an issue when using this module under CentOS7: because `python-crypto` package has been replaced by `python2-crypto`, the package provider continuously attempts to 'reinstall' the package, and issues a carbon restart on each run as a result:
```
Notice: /Stage[main]/Graphite::Install/Package[python-crypto]/ensure: created
Info: Class[Graphite::Install]: Scheduling refresh of Class[Graphite::Config]
Info: Class[Graphite::Install]: Scheduling refresh of Exec[Initial django db creation]
Info: Class[Graphite::Config]: Scheduling refresh of Exec[Initial django db creation]
Info: Class[Graphite::Config]: Scheduling refresh of Service[carbon-cache]
Info: Class[Graphite::Config]: Scheduling refresh of Service[carbon-relay]
Notice: /Stage[main]/Graphite::Config/Exec[Initial django db creation]: Triggered 'refresh' from 2 events
Notice: /Stage[main]/Graphite::Config/Service[carbon-cache]: Triggered 'refresh' from 1 events
Notice: /Stage[main]/Graphite::Config/Service[carbon-relay]: Triggered 'refresh' from 1 events
```

Package details:
```
[root@user-infradev-graphite-eu-west-1b ~]# yum info python-crypto
Loaded plugins: fastestmirror, s3iam
Loading mirror speeds from cached hostfile
 * base: ftp.heanet.ie
 * epel: s3-mirror-eu-west-1.fedoraproject.org
 * extras: ftp.heanet.ie
 * updates: ftp.heanet.ie
Available Packages
Name        : python-crypto
Arch        : x86_64
Version     : 2.6.1
Release     : 1.el7.centos
Size        : 470 k
Repo        : extras/7/x86_64
Summary     : Cryptography library for Python
URL         : http://www.pycrypto.org/
Licence     : Public Domain and Python
Description : PyCrypto is a collection of both secure hash functions (such as MD5 and
            : SHA), and various encryption algorithms (AES, DES, RSA, ElGamal, etc.).

[root@user-infradev-graphite-eu-west-1b ~]# yum install python-crypto
Loaded plugins: fastestmirror, s3iam
artifactory_puppet_stable                                                                                                                                                                                              | 1.3 kB  00:00:00
artifactory_puppet_unstable                                                                                                                                                                                            | 1.3 kB  00:00:00
Loading mirror speeds from cached hostfile
 * base: ftp.heanet.ie
 * epel: s3-mirror-eu-west-1.fedoraproject.org
 * extras: ftp.heanet.ie
 * updates: ftp.heanet.ie
Package python-crypto-2.6.1-1.el7.centos.x86_64 is obsoleted by python2-crypto-2.6.1-9.el7.x86_64 which is already installed
Nothing to do
```

**Comment by dwerder** [comment]
Which repos do u use?

**Comment by madAndroid** [comment]
We've got the base centos repos (including extras) and EPEL enabled .. looks like python2-crypto is coming from EPEL:
```
[root@user-infradev-graphite-eu-west-1b ~]# yum info python2-crypto
Loaded plugins: fastestmirror, s3iam
Loading mirror speeds from cached hostfile
 * base: ftp.heanet.ie
 * epel: s3-mirror-eu-west-1.fedoraproject.org
 * extras: ftp.heanet.ie
 * updates: ftp.heanet.ie
Installed Packages
Name        : python2-crypto
Arch        : x86_64
Version     : 2.6.1
Release     : 9.el7
Size        : 2.2 M
Repo        : installed
From repo   : epel
Summary     : Cryptography library for Python 2
URL         : http://www.pycrypto.org/
Licence     : Public Domain and Python
Description : PyCrypto is a collection of both secure hash functions (such as MD5 and
            : SHA), and various encryption algorithms (AES, DES, RSA, ElGamal, etc.).
            :
            : This is the Python 2 build of the package.
```

**Comment by dwerder** [comment]
Fixed by #247
